### PR TITLE
Wordpress - Container Image

### LINKED ISSUE #7: Wordpress - Container Image
**Objectives:**
- Create a docker image with wordpress
- Push the image to ECR Repository created on #2

**Acceptance Criteria:**
- [x] The Dockerfile is not using latest tag
- [x] The documentation is updated with instruction on how to create the docker image
- [x] Image is being tagged with the commit SHORT_SHA
 
Todo:
- [x] Create wordpress docker image
- [x] Create make file to build / push image

### PR DESCRIPTION
**Pull request by alanlima**
## Work tracking list

- [x] Wordpress docker image
  - [x] do not use the latest tag in the base wordpress image
- [x] **Makefile** with operations:
  - [x] **build**: create a new docker image
  - [x] **push**: tag the latest image built with the commit sha and push it to AWS ECR
    - [x] image tagged with the commit SHORT_SHA
  - [x] **ecr-login**: get the login password from ecr to authenticate with the `docker login`
  - [x] **deploy**: execute the following operations in order: **ecr-login, build, push**
- [x] **docker-compose** file for local tests and to define the building parameters to build the docker image.
- [x] **Documentation**: document all the operations and environment variables used.

### CODE REVIEW

**Review by fabioacp — COMMENTED** [review]
Could you please squash the commits? It's a best practice and I think that for our project we can aim to have one commit per PR. :)

**Comment by alanlima** [comment]
> Could you please squash the commits? It's a best practice and I think that for our project we can aim to have one commit per PR. :)

Yeah, I am doing that but I cannot do that on my own PR..

This one os ready to be reviewed and once approved the approver should squash and merge 😁

**Comment by caiocezart** [comment]
Ideally, the PR owner should be the one that actually merge the code in.
The reason being that only the owner or someone that fully understand the new code will know if the code is ready to be merged or not

File: docker/Makefile
Comment (fabioacp) [inline docker/Makefile:11]: Great work Alan, everything worked just fine. Just a very small detail.
I think the reset variable should be something like `C_RESET ?= \033[0m`, shouldn't it?

File: README.md
Comment (fabioacp) [inline README.md]: I think you can discard the changes in the README file. We can add all these references later.

**Review by fabioacp — CHANGES_REQUESTED** [review]

File: docker/Makefile
Comment (alanlima) [reply docker/Makefile:11]: Thanks @fabioacp 
Both way represents the <ESC> characters, but I should have used a single pattern instead of the alias.
I normalise them to use `\e` for both variables. I just choice `\e` instead of `\033` for clarity, let me know if you think otherwise so I can use `\033` instead. 🤗

**Review by alanlima — COMMENTED** [review]
Thanks @fabioacp for your review.

I forgot about the `readme.md` file 😬. I removed it.

File: README.md
Comment (alanlima) [reply README.md]: Done, thanks ;)

**Review by fabioacp — DISMISSED** [review]

File: docker/Makefile
Comment (fabioacp) [reply docker/Makefile:11]: no worries. Great job mate!

**Comment by alanlima** [comment]
Hey @fabioacp,
I just rebase this branch from master to update with the latest changes and squash the commits.
Could you please check it and approve if everything looks good for you? 😁

**Review by fabioacp — APPROVED** [review]
Nice job!👍

**Pull request merged**
